datumaro/plugins/imagenet_txt_format.py

```diff
@@ -87,17 +102,34 @@ def build_cmdline_parser(cls, **kwargs):
         parser = super().build_cmdline_parser(**kwargs)
         parser.add_argument('--labels-file', dest='labels',
             help="Path to the file with label descriptions (synsets.txt)")
+        parser.add_argument('--no-labels-file', dest='labels',
+            action='store_const', const=(),
+            help="Do not load a file with label descriptions")
+        parser.add_argument('--auto-labels', action='store_true',
+            help="Automatically create generic label descriptions "
+                "if an annotation refers to a label "
+                "for which no description exists")
```

Comment:
```suggestion
            help="Use auto-generated labels for annotations, "
                "when no label information is available in the dataset")
```

Reply:
That doesn't describe the option correctly. It has an effect even if label information _is_ available.

Reply:
How it can be used in such cases?